cms/README.md

```diff
@@ -15,8 +21,9 @@ Start strapi:
 ```
 git clone https://github.com/rocketchat/RC4Community
 cd cms
-npm i
-INITIALIZE_DATA=true npm run develop
+yarn install
+yarn build
```

Comment:
Why do we need `yarn build` ?

Reply:
The Strapi docs mentioned, but it runs perfectly fine without this also
<img width="626" alt="image" src="https://user-images.githubusercontent.com/61188295/171637392-4e287186-555b-4030-b1a1-6fda52c8d17b.png">